.travis.yml

```diff
@@ -33,21 +27,66 @@ branches:
     - /^amp-release-.*$/
     - /^revert-.*$/
 addons:
-  chrome: stable
   hosts:
     - ads.localhost
     - iframe.localhost
     # Requested by some tests because they need a valid font host,
     # but should not resolve in tests.
     - fonts.googleapis.com
-  apt:
-    packages:
-      - protobuf-compiler
-      - python-protobuf
-matrix:
+stages:
+  - build
+  - test
+jobs:
+  fast_finish: true
+  allowed_failures:
+    - stage: test
+      name: "Remote (Sauce Labs) Tests"
+      script:
+        - node build-system/pr-check/remote-tests.js
+      after_script:
+        - build-system/sauce_connect/stop_sauce_connect.sh
```

Comment:
For easy maintainability, move this section to the bottom, so that when we make the Sauce Labs job blocking once again, all one has to do is remove the `allowed_failures` header.